tests/sync/sync_test_utils.cpp

```diff
@@ -56,13 +56,10 @@ bool results_contains_original_name(SyncFileActionMetadataResults& results, cons
 }
 
 std::string tmp_dir() {
-    const char* dir = getenv("TMPDIR");
```

Comment:
Why is this switching away from using $TMPDIR? Being able to override the temporary directory in the standard manner is useful.